api/src/chat/schemas/types/message.ts

```diff
@@ -312,11 +312,6 @@ export const validMessageTextSchema = z.object({
   message: z.string(),
```

Comment:
Hi @abdou6666, i notice that the message validation fix (merged in the main branch) is not included in this PR #741
![image](https://github.com/user-attachments/assets/222b26f0-1e59-40fb-bafd-66f1a19c230b)

feat: refactor zod validation #739
![image](https://github.com/user-attachments/assets/4a34a0dd-0f61-4ad3-aad0-789ca7914d7d)